packages/icon/src/IconBase.ts

```diff
@@ -33,7 +33,6 @@ export class IconBase extends SpectrumElement {
     public static override get styles(): CSSResultArray {
         return [iconStyles];
     }
-
```

Comment:
Did you mean to remove this whitespace? It seems useful to delineate the two properties.

Reply:
Ah! I was testing the linting with some additional fail checks here. Should be removed. Thanks for catching this